portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,285 @@
 <!DOCTYPE html>
-<html>
-  <head>
+<html lang="en">
+
+<head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Toch's Portfolio</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+</head>
+
+<body>
+
+    <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
+        <a class="navbar-brand" href="#">Toch's Portfolio</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup"
+            aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+            <div class="navbar-nav">
+                <a class="nav-item nav-link" href="#about">About</a>
+
+                <a class="nav-item nav-link" href="#projects">Projects</a>
+            </div>
+        </div>
+    </nav>
+
+
+
+    <section id="landing" class="section">
+
+        <div class="container">
+            <img src="/images/landing_pic.jpeg" alt="Tochukwu Ibe-Ekeocha">
+            <h1>Tochukwu Ibe-Ekeocha</h1>
+            <h3>Software Developer, Enterprenure, Tech Enthusiast</h3>
+        </div>
+
+    </section>
+
+    <section id="about" class="section ">
+
+        <div class="container">
+            <h1 class="section-title">About</h1>
+
+            <div class="row flex-cads">
+
+                <div class="col">
+                    <div class="card" style="width: 15rem;">
+                        <div class="card-body card-content">
+                            <h3 class="card-title">Personal</h3>
+                            <ion-icon class="icon" name="person"></ion-icon>
+                            <button id="personal-btn" onclick="personal_modal.style.display = 'flex'" type="button"
```

Comment:
Nice and simple 'modal' dialog implementation!

A small improvement would be to organize the DOM manipulation into functions in your `script.js`, e.g.

```
function showModal(name) {
  document.getElementById(name)....
}
function closeModal(..) {
}
```

It makes it easier to add functionality later without having to fix it in so many places.